docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -992,4 +991,169 @@ eclagent:
       </sect3>
     </sect2>
   </sect1>
+
+  <sect1 id="LDAPSecurity_forConatiners">
+    <title>LDAP Security for Containers</title>
+
+    <para>Containerized LDAP support for HPCC Systems is accomplished by
+    externalizing the LDAP Active Directory Security Manager administrators
+    account credentials using Kubernetes or Hashicorp Vault secrets. As a
+    prerequisite to use these externalized credentials, you should be familiar
+    with setting up Kubernetes and/or Hashicorp Vault secrets.</para>
+
+    <para>The LDAP Administrator account is the only HPCC user account that
```

Comment:
This is not the LDAP Administrator account. It's the HPCC Admin account. It does not have Active Directory administrator rights. It has must have admin rights to all of the Base DNs used by the platform. I would not bother mentioning the actual BaseDNs here. That is another topic. I would just concentrate on what needs to be done to secure the credentials for the HPCC Admin user. In fact I wouldn't mention what rights is must have (that should all be covered elsewhere and a pointer to how the HPCC Admin user must be configured may be appropriate here)